docs/data-sources/dms_kafka_flavors.md

```diff
@@ -0,0 +1,153 @@
+---
+subcategory: "Distributed Message Service (DMS)"
+---
+
+# huaweicloud_dms_kafka_flavors
+
+Use this data source to get the list of available flavor details within HuaweiCloud.
+
+## Example Usage
+
+### Query the list of kafka flavors that support the prepaid charging mode
+
+```hcl
+data "huaweicloud_dms_kafka_flavors" "test" {
+  charging_mode = "hourly"
```

Comment:
don't use as the first example because we don't support prepaid in stance for now

Reply:
done.